src/System.Net.Security/tests/FunctionalTests/DummyTcpServer.cs

```diff
@@ -140,40 +143,25 @@ private void OnAccept(IAsyncResult result)
                 if (useSsl)
                 {
                     state = new ClientState(client, sslEncryptionPolicy);
-                    Logger.LogInformation("Server: starting SSL authentication.");
+                    _log.WriteLine("Server: starting SSL authentication.");
 
 
                     SslStream sslStream = null;
                     X509Certificate2 certificate;
+                    certificate = new X509Certificate2("DummyTcpServer.pfx");
```

Comment:
I think that right now the Unix version incorrectly always de-prioritizes the cert with the private key (trying to match the order that Windows seemed to be writing/reading a full chain for a collection).  Using the collection and finding the one which says HasPrivateKey=>true is definitely the most stable approach for a multi-cert PFX.